spec/api.js

```diff
@@ -0,0 +1,5 @@
+require( 'dotenv' ).config();
+const controllers = require('../server/controllers.js');
+
```

Comment:
not sure i understand the naming convention of spec for the file. Is the intention to make of of these files for each route or to do all gets in this file for all routes?